GitHub_stuff.md

```diff
@@ -0,0 +1,7 @@
+# Github stuff we learned these two days
+
+First of all the MD tutorial is available: [Here](https://www.markdowntutorial.com)
+Also, Praveen has been nice to provide us his tutorials and cheatsheets.
+
+*Github commands: [Here](https://blog.praveen.science/git-shortcuts/)
+*Contributing in a repository tutorial: [Here](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md)
```

Comment:
Missed a space after `*`. Check the preview on how it looks.

Reply:
ok, done